cdap-docs/vars

```diff
@@ -27,11 +27,12 @@ BUILD_RST_HASH_LOCATION="cdap-docs/vars"
 BUILD_RST_HASH="039fc5b9a1decc5c67f17c332413f5d9"
 
 # This needs to be explicitly set as Git has trouble identifying it.
-GIT_BRANCH_PARENT="develop"
+GIT_BRANCH_PARENT="release/3.5"
 
 # Used by cdap-apps for Hydrator documentation
-GIT_BRANCH_CASK_HYDRATOR="develop"
+GIT_BRANCH_CASK_HYDRATOR="release/1.4"
+GIT_BRANCH_CASK_TRACKER="release/0.2"
 
 # Used by developers-manual
-CDAP_CLIENTS_RELEASE_VERSION="1.2.0"
+CDAP_CLIENTS_RELEASE_VERSION="1.3.0"
```

Comment:
No. I am saying that we converted it to a Gem for release to Rubygems after 1.2 and we only have a 1.3.0.a pre-release on Rubygems, which is the latest, which isn't usable in Chef. Let's just ignore this. I was simply confirming that there isn't a 1.3 release (yet).